lib/winston/logger.js

```diff
@@ -12,7 +12,7 @@ const asyncForEach = require('async/forEach');
 const { LEVEL, SPLAT } = require('triple-beam');
 const isStream = require('is-stream');
 const ExceptionHandler = require('./exception-handler');
-const LegacyTransportStream = require('winston-transport/legacy');
+const LegacyTransportStream = require('winston-transport').LegacyTransportStream;
```

Comment:
Why change this?

Reply:
Please see https://github.com/winstonjs/winston/pull/1418#discussion_r215422536